.github/workflows/test_linux_sbc.yml

```
@@ -149,7 +149,8 @@ jobs:
             bebop,
             erlebrain2,
             pxfmini,
-            pxf
+            pxf,
+            canzero
```

```suggestion
            canzero,
```

... to simplify future diffs